doc/guides/contributing/pull-requests.md

```diff
@@ -579,6 +579,8 @@ If you want to know more about the code review and the landing process, see the
 * `test`
 * `tools`
 
+You can find the full list of supported subsystems in the
+[nodejs/core-validate-commit][] repo.
```

Comment:
```suggestion
[nodejs/core-validate-commit][] repository.
```

See b9866098a7f8588970bb9107eadd6deb853c1fa2, b11d3f15b31e7a39d3f6a6e0d3f59f021144e813, 4de6f207dca12164b9403e4d539b843d4d21f820, and bbdc15bbdbb61f4e7d6f388b93ea32175a305bd3.